app/exceptions/ExerciseCompilationSoftException.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Exceptions;
+
+use Nette\Http\IResponse;
+
+/**
+ * Exception used in exercise compilation to job configuration.
+ * Used if user inputs are incorrect or invalid (e.g. uploaded files).
+ */
+class ExerciseCompilationSoftException extends ExerciseCompilationException {
+
+  /**
+   * Constructor.
+   * @param string $msg description
+   */
+  public function __construct(string $msg = 'Please contact system administrator') {
```

Comment:
This message is awkward. The soft error is presumably an user error, so contacting admin should not be a first choice. What about something like "Please, check the exercise instructions."